verification/signature_verification_levels.go

```diff
@@ -0,0 +1,105 @@
+package verification
+
+import "fmt"
+
+// VerificationType is an enum for signature verification types such as Integrity, Authenticity, etc.
+type VerificationType string
+
+// VerificationAction is an enum for signature verification actions such as Enforced, Logged, Skipped.
+type VerificationAction string
+
+// SignatureVerificationLevel encapsulates the signature verification preset and it's actions for each verification type
+type SignatureVerificationLevel struct {
+	Name            string
+	VerificationMap map[VerificationType]VerificationAction
+}
+
+const (
+	Integrity             VerificationType = "Integrity"
+	Authenticity          VerificationType = "Authenticity"
+	TrustedTimestampCheck VerificationType = "TrustedTimestampCheck"
+	ExpiryCheck           VerificationType = "ExpiryCheck"
+	RevocationCheck       VerificationType = "RevocationCheck"
+
+	Enforced VerificationAction = "Enforced"
+	Logged   VerificationAction = "Logged"
+	Skipped  VerificationAction = "Skipped"
+)
+
+var (
+	Strict = SignatureVerificationLevel{
```

Comment:
Should we have pointers instead of instances

```suggestion
	Strict = &SignatureVerificationLevel{
```

Reply:
Don't see why we can't do that